lighthouse-core/audits/seo/meta-description.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../audit');
+
+class Description extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Content Best Practices',
+      name: 'meta-description',
+      description: 'Document has a meta description',
+      failureDescription: 'Document does not have a meta description',
+      helpText: 'Meta descriptions may be included in search results to concisely summarize ' +
+          'page content. Read more in the ' +
+          '[Search Console Help page](https://support.google.com/webmasters/answer/35624?hl=en#1).',
+      requiredArtifacts: ['MetaDescription']
+    };
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    if (artifacts.MetaDescription === null) {
+      return {
+        rawValue: false,
+        debugString: 'Description meta tag is missing.'
```

Comment:
Actually now that you've brought it up in the other issue, I think I might have a slight preference to put these in the `displayValue` and keep `debugString` for errors like the gatherer failing (which is handled automatically)

Reply:
> Actually now that you've brought it up in the other issue, I think I might have a slight preference to put these in the displayValue and keep debugString for errors like the gatherer failing (which is handled automatically)

agree about confusion between user vs system errors in `debugString`, but don't we still want the red string for failure here? (We still use the guidance "Optional error string for helping the user figure out why they failed here." in our docs)

Reply:
IMO `debugString` is more readable and, as @brendankenny mentioned, in this case it matches the description in the docs:

https://github.com/GoogleChrome/lighthouse/blob/fadaef95137181aa41ed44fd78f17d3a8b85a2c3/lighthouse-core/closure/typedefs/AuditResult.js#L31-L35

@patrickhulce WDYT? Can we keep the `debugString` here?

Reply:
Yeah sgtm, I'm sort of off the reservation when it comes to my hopes for debugString anyway :)

Reply:
I'll put the title and debugString in the order you'd see them in the report:

> #### Document does not have a meta description
> Description meta tag is missing

Given that the debugString duplicates so much, i think we should exclude it.